build/Makefile

```diff
@@ -501,7 +505,11 @@ testrun : test
 TEST_STATUS:
 	@echo "========================= Performing Building provider tests"
 
-test : TEST_STATUS $(SCXPAL_INTERMEDIATE_DIR) $(INTERMEDIATE_DIR)/testrunner
+TEST_DATA_PREP:
+	@echo "========================= Performing Test data preparation"
+	cp -R $(TEST_DIR)/omi_preexec/testdata/cnfs $(INTERMEDIATE_DIR)/.
```

Comment:
you mean via a soft link instead of a copy, correct?

Just to be on same page, the paths in the includedirs and include in all the files will be affected.